content/src/content/jcr_root/apps/core/wcm/components/modal/v1/modal/modal.html

```diff
@@ -0,0 +1,28 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe Systems Incorporated
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<div data-sly-use.modal="com.adobe.cq.wcm.core.components.models.Modal" data-sly-use.template="core/wcm/components/commons/v1/templates.html">
+    <div data-sly-test="${wcmmode.edit}">
+        <h4>Modal Id: ${modal.modalId}</h4>
+        <h4 data-sly-test="${modal.description}">Description: ${modal.description}</h4>
+        <h4 data-sly-test="${modal.fragmentType == 'cf' && modal.contentFragmentPath}"> Content Path: ${modal.contentFragmentPath}</h4>
+        <h4 data-sly-test="${modal.fragmentType == 'xf' && modal.experienceFragmentPath}"> Content Path: ${modal.experienceFragmentPath}</h4>
+
+    </div>
+    <div data-sly-test="${!wcmmode.edit}">
+        <div class="cmp_modal" id="${modal.modalId}" data-modal-show="${properties.showModalByDefault}" tabindex="-1" role="dialog" aria-hidden="true" data-content-url="${modal.fragmentType == 'cf' ? modal.contentFragmentPath : modal.experienceFragmentPath}">

```

Comment:
Does tingle manage these aria attributes? From what I can tell it doesn't, or maybe I just missed it. If not, we shouldn't add this `aria-hidden` attribute as it will always be incorrect. Also highlights the problem with relying on a 3rd party widget for the UI handling, we won't be able to claim that this component is accessible.